neural_speed/convert/convert_llama.py

```diff
@@ -1089,6 +1089,8 @@ def write_file_header(self, params: Params, file_type: NEFileType) -> None:
         self.fout.write(struct.pack("i", params.ffn_hidden_size))
         self.fout.write(struct.pack("i", 0))
 
+        self.fout.write(struct.pack("i", 0))
+        self.fout.write(struct.pack("i", 0))
```

Comment:
please also add these parameters in  convert_quantized_llama.py and other convert_quantized_xxxx.py files.

Reply:
done